pkg/controller/petset/pet.go

```diff
@@ -216,6 +220,7 @@ func (p *apiServerPetClient) Update(pet *pcb, expectedPet *pcb) (updateErr error
 		if p, getErr = pc.Get(pod.Name); getErr != nil {
 			return getErr
 		}
+		time.Sleep(retrySleepTime)
```

Comment:
> if this fails we'll just enqueue into a delay queue which will impose exponential build up of delay.

Sorry, I am afraid I can't find the delay queue you mentioned.

The purpose of sleep between `Update()` retries is just want to reduce the pressure of API Server. Because if we `Get(pod.Name)` successfully but `Update(p)` failed, probably API Server is under pressure.

I just refer to: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/node/nodecontroller.go#L536

If I missed something, please point it out. Thanks!